README.md

```diff
@@ -0,0 +1,95 @@
+# UrContacts
+> A contact list application
+
+UrContacts is a React contact list app built with full CRUD capabilities. It uses json-server to locally store your data in a JSON file at the backend. Because your data persists on a locally hosted database, your data is completely private.
+
+## Installation and Build
+
+1. Clone GitHub repo into a desired local root directory: 
+    - HTTPS: https://github.com/dyazdani/contact-list.git
+    - SSH: `git@github.com:dyazdani/contact-list.git`
+    - GitHub CLI: `gh repo clone dyazdani/contact-list` 
```

Comment:
You don't need to say how to clone, someone using GitHub will typically know how to do this. And this way if you ever change your username or this repo's name, it won't need to be kept up to date.

Also noting that they can choose to download instead of cloning if they want.

```suggestion
1. Clone or download GitHub repo into a desired local root directory.
```

Reply:
Got it. Out of curiosity, downloading the code would just give yo a copy of it without putting it in a Git repo, right?